### PR TITLE
Fix eap-migration-demo in Olympus

### PR DESCRIPTION
**Pull request by rpelisse**
Issue: https://issues.redhat.com/browse/AMW-278
Harmonia PR: https://github.com/jboss-set/harmonia/pull/134

### CODE REVIEW

File: molecule/default/molecule.yml [hunk 1]
```diff
@@ -23,6 +23,9 @@ platforms:
       - 0.0.0.0:8080:8080/TCP
 provisioner:
   name: ansible
+  env:
+    PROD_JBOSSNETWORK_API_CLIENTID: "${PROD_JBOSSNETWORK_API_CLIENTID}"
+    PROD_JBOSSNETWORK_API_SECRET: "${PROD_JBOSSNETWORK_API_SECRET}"
```
Comment (rpelisse) [inline molecule/default/molecule.yml]: @RanabirChakraborty I don't think you need that. This part of the Molecule file is to **add** env vars that are not already defined. While it could feel clear to add these here, I think it's better to hide this complexity from the demo code. (especially as user will probably provides those creds using the -e @file.yml option anyway).

**Comment by RanabirChakraborty** [comment]
@rpelisse Now it's successfully running on Olympus - ansible-eap-migration-demo/62, Please review it.

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -23,6 +23,8 @@ platforms:
       - 0.0.0.0:8080:8080/TCP
 provisioner:
   name: ansible
+  env:
+    ANSIBLE_VERBOSITY: 3
```
Comment (rpelisse) [inline molecule/default/molecule.yml]: @RanabirChakraborty I think this needs to be removed, no ?

File: playbook.yml [hunk 3]
```diff
@@ -2,13 +2,3 @@
 - import_playbook: setup_eap.yml
 - import_playbook: prepare_migration.yml
 - import_playbook: perform_migration.yml
-
```
Comment (rpelisse) [inline playbook.yml:5]: Why did you remove the 7.3 to 7.4 demo?

**Review by rpelisse — COMMENTED** [review]
See comment in files.

File: playbook.yml (hunk 3 above)
Comment (RanabirChakraborty) [reply playbook.yml:5]: we are already doing that in our demo from EAP 7.3 to EAP 7.4, so removed that part.

File: molecule/default/molecule.yml (hunk 2 above)
Comment (RanabirChakraborty) [reply molecule/default/molecule.yml]: fixed it now!
